### PR TITLE
(Fixed #25) Add run_python launcher script

### LINKED ISSUE #25: Add run_python launcher script
* One of the input locations is the python application (or specify an S3 URL to download)
* Creates a virtualenv and pip installs the requirements (if they exist)
* Takes python script and arguments to run

### PR DESCRIPTION
**Pull request by sethyates**

### CODE REVIEW

File: scripts/deploy-hyperion-emr-env.sh [hunk 1]
```diff
@@ -3,18 +3,16 @@
...
-    echo "Usage: $0 <env-file-s3-path>"
-    exit 1
+  echo "Usage: $0 <env-file-s3-path>"
+  exit 1
 fi
 
 S3_ENV_FILE=$1; shift
 ENV_FILE="/home/hadoop/hyperion_env.sh"
 
-if [[ -f $ENV_FILE ]]; then
```
Comment (sethyates) [inline scripts/deploy-hyperion-emr-env.sh]: This check was redundant if using `-f` on `rm`.

File: scripts/configure-emr.sh [hunk 2]
```diff
@@ -1,7 +1,7 @@
 #! /usr/bin/env bash
 
-# Usage: $0 envrionments e.g. $0 ORG_ENV=prod
+# Usage: $0 environments e.g. $0 ORG_ENV=prod
 
 for henv in $@; do
-    echo "export $henv" >> /home/hadoop/hyperion_env.sh
+  echo "export $henv" >> /home/hadoop/hyperion_env.sh
```
Comment (realstraw) [inline scripts/configure-emr.sh]: are we changing shell scripts to 2-space indentation?

Comment (sethyates) [inline scripts/configure-emr.sh]: It's just my setup. Should I revert?

Comment (realstraw) [inline scripts/configure-emr.sh]: I was using 4 for shell scripts. Either is good though. Cannot find a widely used standard online. Google (https://google-styleguide.googlecode.com/svn/trunk/shell.xml#Indentation) says 2.

Comment (sethyates) [inline scripts/configure-emr.sh]: Sounds good to me ;-)

File: scripts/run-python.sh [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+  fi
+fi
+
+# If a requirements file exists, then setup a virtual environment
+# and install the requirements into that
+if [ -f ${REQUIREMENTS} ]; then
+  echo "Setting up virtual environment"
+  sudo easy_install virtualenv
+
+  virtualenv -v .
```
Comment (realstraw) [inline scripts/run-python.sh]: Should we use a special folder instead of `.`. Something like `.ENV` or `~/.ENV`?

File: scripts/run-python.sh [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+      exit 3
+    fi
+  fi
+fi
+
+# If a requirements file exists, then setup a virtual environment
+# and install the requirements into that
+if [ -f ${REQUIREMENTS} ]; then
+  echo "Setting up virtual environment"
+  sudo easy_install virtualenv
```
Comment (realstraw) [inline scripts/run-python.sh]: `pip` instead of `easy_install`?

Comment (sethyates) [inline scripts/run-python.sh]: That'd be nice, hey. Alas, Pip isn't available on the AMI....

File: scripts/run-python.sh (hunk 3 above)
Comment (sethyates) [inline scripts/run-python.sh]: Sure. Will change.

File: scripts/run-python.sh (hunk 4 above)
Comment (realstraw) [inline scripts/run-python.sh]: Ah, ok.

**Comment by realstraw** [comment]
:+1:

**Pull request closed without merging**